src/allotropy/parsers/appbio_quantstudio/appbio_quantstudio_structure.py

```diff
@@ -248,30 +248,32 @@ class AmplificationData:
     rn: list[float | None]
     delta_rn: list[float | None]
 
-    @staticmethod
-    def create(reader: LinesReader) -> dict[int, dict[str, AmplificationData]]:
-        assert_not_none(
-            reader.drop_until(r"^\[Amplification Data\]"),
-            msg="Unable to find 'Amplification Data' section in file.",
-        )
 
-        reader.pop()  # remove title
-        lines = list(reader.pop_until(r"^\[.+\]"))
-        csv_stream = StringIO("\n".join(lines))
-        data = read_csv(csv_stream, sep="\t", thousands=r",")
+def create_amplification_data(
```

Comment:
Note to reviewer: this allows re-use of `AmplificationData` in design and analysis parser